File: BEE 1037.java
import java.io.IOException;
import java.util.Scanner;

public class Main{
    public static void main(String[] args) throws IOException{
        Scanner input = new Scanner(System.in);
        float num = input.nextFloat();
        if (num > 75.00001 && num <= 100){
            System.out.println("Intervalo (75,100]");
        }
        else if (num > 50.00001 && num <= 75){
            System.out.println("Intervalo (50,75]");
        }
        else if (num > 25.00001 && num <= 50){
            System.out.println("Intervalo (25,50]");
        }
        else if (num >= 0 && num <= 25){
            System.out.println("Intervalo [0,25]");
        }
        else{
            System.out.println("Fora de intervalo");
        }
    }
}

File: BEE 1037.cpp
#include <iostream>

using namespace std;

int main(){
    float num;
    cin >> num;

    if (num > 75.00001 && num <= 100){
        cout << "Intervalo (75,100]" << endl;
    }
    else if (num > 50.00001 && num <= 75){
        cout << "Intervalo (50,75]" << endl;
    }
    else if (num > 25.00001 && num <= 50){
        cout << "Intervalo (25,50]" << endl;
    }
    else if (num >= 0 && num <= 25){
        cout << "Intervalo [0,25]" << endl;
    }
    else{
        cout << "Fora de intervalo" << endl;
    }

    return 0;
}